src/System Application/App/User Details/src/UserDetails.Page.al

```diff
@@ -0,0 +1,105 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Security.User;
+
+using System.Environment;
+
+/// <summary>
+/// Shows detailed user information, such as unique identifiers, information about permission sets etc.
+/// </summary>
+page 774 "User Details"
+{
+    AboutText = 'View the additional information about users in a list view, which allows for easy searching and filtering.';
+    AboutTitle = 'About the users detailed view';
+    ApplicationArea = Basic, Suite;
+    Caption = 'Users';
+    DeleteAllowed = false;
+    Editable = false;
+    InsertAllowed = false;
```

Comment:
No need for InsertAllowed = false, ModifyAllowed = false and DeleteAllowed = false when you set Editable = false. These are automatically set to false in this case. Only needed if you don't set Editable = false and want special logic.

Reply:
Editable = false can be overwritten by PageExtensions.

InsertAllowed and the other properties aren't changeable by PageExtensions.

Since this page is extensible I'd vote for ensuring that no extension should be able to make changes, that users can insert any value on this page.